boards/nrf51dk/jtag/flash-full.jlink

```diff
@@ -0,0 +1,6 @@
+r
+loadfile target/nrf51/release/nrf51dk-blink.hex
```

Comment:
I think this might be a divergence between MBED and non-MBED platforms. MBED platforms provide a really nice and simple programming interface (copy to the disk image) which doesn't require JTAG or serial support. I'd rather that we don't require people install JTAG to program the device.